### ISSUE TITLE
Request for a "Default Hardware Acceleration(NVENC instead of none)" option in the settings (for available formats)

### ISSUE THREAD

**Issue by detfusco** [paulpacifico/shutter-encoder#214]
Every single time I open and close Shutter Encoder, my preference of hardware acceleration gets resetted to none. I forgot and suffered the consequences of my actions far too many times because of that silly lil option was set to none. If there was an option in the settings for making NVENC the default instead of none, I'd do it in a heartbeat. And it would be a net benefit for the users.

Anyway, thanks for best encoder/converter there is across all OSes, you're the doing the lord's work :) 👍

**Comment by paulpacifico** [comment]
Use the preset from the star icon next to '**Start function**' button and save your settings.

You can load it at the startup from the top left cog icon '**load preset at startup**'.

I will not make it by default because the quality is lower than CPU only.


Thanks for your feedback 🙏
Paul.

**Comment by detfusco** [comment]
I wasn’t talking about making Hardware acceleration being on by default, I was just asking for the ability to change the default option(a dropdown menu). Load preset at startup is also a confusing name. I thought that meant windows startup instead, so I dismissed it thinking it would start the program upon windows logon. 
Anyway I did the preset thing now and it seems to work but I still think that’s a complicated solution to a simple problem :S

Thanks for your swift reply :)
